code/__HELPERS/game.dm

```diff
@@ -477,7 +477,7 @@
 			if(!gametypeCheck.age_check(M.client))
 				continue
 		if(jobbanType)
-			if(jobban_isbanned(M, jobbanType) || jobban_isbanned(M, "Syndicate"))
+			if(jobban_isbanned(M, jobbanType) || antag_banned(M))
 				continue

```

Comment:
if jobbantype is not set, it should still check softban

Reply:
Done.